less/modals.less

```diff
@@ -0,0 +1,37 @@
+//
+// Modals
+// --------------------------------------------------
+
+
+// Actual modal
+.modal-content {
+  border-top: 4px solid @brand-primary; //old browsers fallback (ie8 etc)
```

Comment:
This line (and comment) can be removed since it's now identical to the one below.